From 66565a3870dbaaed09163f85d0f12d31728b640c Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Tue, 3 Mar 2020 12:39:20 -0800
Subject: [PATCH] Omit tncc python wrapper entirely (juniper vpns? could be
 old).

---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 4fcefee..5c8570e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -137,7 +137,7 @@ EXTRA_DIST += $(shell cd "$(top_srcdir)" && \
 
 DISTCLEANFILES = $(pkgconfig_DATA)
 
-pkglibexec_SCRIPTS = trojans/csd-post.sh trojans/csd-wrapper.sh trojans/tncc-wrapper.py \
+pkglibexec_SCRIPTS = trojans/csd-post.sh trojans/csd-wrapper.sh \
 	trojans/hipreport.sh trojans/hipreport-android.sh
 
 # main.c includes version.c
-- 
2.25.1

